### PR TITLE
Code tidy

### PR DESCRIPTION
**Pull request by moloko**

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]
👁

File: js/adapt-contrib-bookmarking.js [hunk 1]
```diff
@@ -65,47 +67,49 @@ define([
...
-                    yes: "Yes",
-                    no: "No"
+                    yes: 'Yes',
+                    no: 'No'
                 };
             }
-            if (!courseBookmarkModel._buttons.yes) courseBookmarkModel._buttons.yes = "Yes";
-            if (!courseBookmarkModel._buttons.no) courseBookmarkModel._buttons.no = "No";
+            if (!courseBookmarkModel._buttons.yes) courseBookmarkModel._buttons.yes = 'Yes';
+            if (!courseBookmarkModel._buttons.no) courseBookmarkModel._buttons.no = 'No';
```
Comment (tomgreenfield) [inline js/adapt-contrib-bookmarking.js]: Suggest losing the if statements and using the OR operator instead:
```js
var buttons = courseBookmarkModel._buttons || { yes: 'Yes', no: 'No' };
```
And later on:
```js
promptText: buttons.yes || 'Yes'
```

File: js/adapt-contrib-bookmarking.js [hunk 2]
```diff
@@ -65,47 +67,49 @@ define([
...
                     }
                 ],
                 _showIcon: true
             };
 
-            if (Adapt.config.get("_accessibility") && Adapt.config.get("_accessibility")._isActive) {
-                $(".loading").show();
-                $("#a11y-focuser").focus();
-                $("body").attr("aria-hidden", true);
+            if (Adapt.config.get('_accessibility') && Adapt.config.get('_accessibility')._isActive) {
```
Comment (tomgreenfield) [inline js/adapt-contrib-bookmarking.js]: Suggest early return:
```js
var accessibility = Adapt.config.get('_accessibility');

if (!accessibility || !accessibility._isActive) {
    Adapt.trigger('notify:prompt', promptObject);
    return;
}
```

File: js/adapt-contrib-bookmarking.js [hunk 3]
```diff
@@ -133,10 +137,10 @@ define([
         setupMenu: function(menuView) {
             var menuModel = menuView.model;
             //set location as menu id unless menu is course, then reset location
-            if (menuModel.get("_parentId")) return this.setLocationID(menuModel.get("_id"));
+            if (menuModel.get('_parentId')) return this.setLocationID(menuModel.get('_id'));
```
Comment (tomgreenfield) [inline js/adapt-contrib-bookmarking.js]: No need for `return`.

File: js/adapt-contrib-bookmarking.js [hunk 4]
```diff
@@ -52,10 +54,10 @@ define([
                 } catch (error) {
                     return;
                 }
-                
+
```
Comment (tomgreenfield) [inline js/adapt-contrib-bookmarking.js]: No need for try...catch in framework 2.2+.

Not even sure `model` needs to be assigned as a variable?

**Review by tomgreenfield — APPROVED** [review]
👁

**Review by oliverfoster — APPROVED** [review]

**Review by NayanKhedkar — APPROVED** [review]

**Review by brian-learningpool — APPROVED** [review]

**Pull request merged**